packages/components/tree/src/model/tree-store.ts

```diff
@@ -289,10 +289,23 @@ export default class TreeStore {
     leafOnly = false,
     checkedKeys: { [key: string]: boolean }
   ): void {
-    const allNodes = this._getAllNodes().sort((a, b) => b.level - a.level)
+    const allNodes = this._getAllNodes().sort((a, b) => {
+      if (a.level > b.level) {
+        return 1
+      }
+      return -1
+    })
```

Comment:
Any different?

Reply:
The ones with level 1 before modification are ranked last, and those with level 1 after modification are ranked first.

Reply:
const allNodes = this._getAllNodes().sort((a, b) => a.level - b.level);
What about this?

Reply:
Looks good, cleaner this way.